src/SpecterWallet.ts

```diff
@@ -282,13 +271,12 @@ export class SpecterWallet {
     })
     const fee = btc2sat(-tx.fee) /* fee is negative */
 
-    const bankOwnerPath = await bankOwnerMediciPath()
-
-    await MainBook.entry(memo)
-      .debit(lndAccountingPath, sats, { ...metadata })
-      .debit(bankOwnerPath, fee, { ...metadata })
-      .credit(bitcoindAccountingPath, sats + fee, { ...metadata })
-      .commit()
+    await ledger.addHotWalletPayment({
+      description: memo,
+      amount: sats,
+      fee,
+      metadata: { hash: txid },
```

Comment:
maybe better to pass the hash directly?

Reply:
instead of wrapping up in metadata

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/398/commits/2f06c347efa47e9a003146ea9aa445f754720e90